rt/rs/security/oauth-parent/oauth2/src/main/java/org/apache/cxf/rs/security/oauth2/filters/JwtAccessTokenValidator.java

```diff
@@ -76,8 +77,7 @@ private AccessTokenValidation convertClaimsToValidation(JwtClaims claims) {
         if (claims.getIssuedAt() != null) {
             atv.setTokenIssuedAt(claims.getIssuedAt());
         } else {
-            Instant now = Instant.now();
-            atv.setTokenIssuedAt(now.toEpochMilli());
+            atv.setTokenIssuedAt(OAuthUtils.getIssuedAt());
```

Comment:
Indeed,  you are right, it should be in seconds, as per https://tools.ietf.org/html/rfc7519
> value representing the number of seconds from 1970-01-01T00:00:00Z UTC until the specified UTC date/time